src/CMakeLists.txt

```diff
@@ -708,7 +708,7 @@ if(MSVC)
 
 	# Flags for C files (sqlite)
 	# /MT = Link statically with standard library stuff
-	set(CMAKE_C_FLAGS_RELEASE "/O2 /Ob2 /MT")
+	set(CMAKE_C_FLAGS_RELEASE "/MT /O2 /Ob2")
```

Comment:
No, it doesn't. I wanted to brin in some order but removed this change right now to simplify this pull.